### ISSUE TITLE
Enhanced Readme

### ISSUE THREAD

**Issue by dzoba** [emkay/nesly-sound#30]
Could you give a little more info in the readme about how this sets up?  I've run the example program, no errors (woo!), but of course also: nothing happened.

**Comment by emkay** [comment]
yeah sure! Sorry about that. This was made very hacky and was moving towards the JS Conf talk. Will try to give detailed steps.

**Comment by dzoba** [comment]
Awesome, thanks!
